include/libdnf5/base/base.hpp

```diff
@@ -134,6 +135,9 @@ class LIBDNF_API Base {
 
     libdnf5::BaseWeakPtr get_weak_ptr();
 
+    /// Returns true if the plugin is successfully added, false otherwise.
+    bool add_plugin(const std::string & config_file_path, libdnf5::plugin::IPlugin & iplugin_instance);
```

Comment:
I think this should have a docstring.

Reply:
Fixed.
I have also added the missing parameter descriptions.